src/pages/service.tsx

```diff
@@ -697,11 +706,13 @@ Thank you very much`;
 export const getStaticProps = async (props: any) => {
   const isGitlab = process.env.NEXT_PUBLIC_REPO_TYPE === "GITLAB";
   
+  const TERMS_TYPES = await import('@opentermsarchive/terms-types');
+  
   return JSON.parse(
     JSON.stringify({
       props: {
         ...props,
-        documentTypes: await getDocumentTypes(),
+        documentTypes: TERMS_TYPES.default || TERMS_TYPES,
```

Comment:
> Why can't we know exactly how to load the data?

Reply:
We know, it was just to make it resilient for futur updates if we change how it is exported as we do not want to maintain this code, but I can simplify it if you prefer?

Reply:
Since we have now locked dependencies versions, such a change would need a voluntary update in the codebase so I don't see much point in future-proofing at this stage. Up to you 🙂